sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/odbc.yaml

```
@@ -183,8 +183,26 @@ provider:
         post:
           # Avoid matching values found on SourceGraph that look like dummy passwords or insertions.
           patternNot:
-            - "^(?i)pass(word|wd)?(?-i)$|^(?i)pwd(?-i)$|^\\[?PLACEHOLDER\\]?$|^%s$|^\\*{3,}$|^\\{\\d*\\}$|^\\#\\{|^\\{{2,}"
+            - "^(?i)pass(word|wd)$"
+            - "^(?i)pwd"
+            - "^\\[?PLACEHOLDER\\]?$"
+            - "^%s$"
+            - "^\\*{3,}$"
+            - "^\\{\\d*\\}$"
+            - "^\\#\\{"
+            - "^\\{{2,}"
+            # Support for full and partial substitution
+            - "\\{+[^}]*\\}++"
```

Why is the closing curly bracket matched possessively? I think it would only make sense to match the part in the middle possessively:
```suggestion
            - "\\{+[^}]++\\}+"
```